save-frontend/src/main/kotlin/org/cqfn/save/frontend/utils/ProjectMenuBar.kt

```diff
@@ -0,0 +1,12 @@
+package org.cqfn.save.frontend.utils
+
+/**
+ * A value for project menu.
+ */
+@Suppress("WRONG_DECLARATIONS_ORDER")
+enum class ProjectMenuBar {
+    RUN,
```

Comment:
May be?
```suggestion
    EVALUATION,
```

Reply:
@akuleshov7 and I decided to leave it like that for now.